internal/cmd/users.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	survey "github.com/AlecAivazis/survey/v2"
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
```

Comment:
I think we need to use the k8s library consistently so that all the commands `yaml` output looks the same. Sorry for the back-and-forth on this.

```suggestion
	"sigs.k8s.io/yaml"
```

This will change the expected output again.